### ISSUE TITLE
Client and server guides seems to be outdated

### ISSUE THREAD

**Issue by z0mbie42** [jmapio/jmap#323]
Hi,

It seems that the implementation guides (https://jmap.io/server.html & https://jmap.io/client.html) are outdated.
For example they use `ModSeq` rather than `state` and `fetchRecordProperties` rather than `properties`.

It's confusing, Is there plan to update them?

If I'm wrong, feel free to close the issue.

**Comment by neilj** [comment]
Yes, they are out of date. We would like to update them, but no one has had the time to do so so far.

**Comment by svenstaro** [comment]
I think it would be quite a good idea to update the guide if you want JMAP to gain traction. Having an outdated guide for longer than two years now doesn't really speak to me as a developer and I'm sure it's not just me.

**Comment by q2dg** [comment]
Moreover, last post written in the web's "News" section dates from 2020...not very exciting.
